superset/config.py

```diff
@@ -300,6 +300,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 DEFAULT_FEATURE_FLAGS: Dict[str, bool] = {
     # Experimental feature introducing a client (browser) cache
     "CLIENT_CACHE": False,
+    "ENABLE_DATASET_SOURCE_EDIT": True,
```

Comment:
nit: Can we name this `DISABLE_DATASET_SOURCE_EDIT` and set it to `False` by default? IMO every feature flag should be false by default, similar to every A/B testing experience starts with a current base (new feature being disabled) vs a new feature. We set to default feature flag to true only when we put an A/B testing full-on, but they will be cleaned up eventually.

Reply:
fixed.